app/src/chat_engine.py

```diff
@@ -402,13 +402,30 @@ class ImagineLaEngine(BaseEngine):
 - Or other kinds of support questions for the Benefit Navigator tool
 then set canned_response to: "To get support with that issue, select "Need help? Contact the support team" at the top of this chatbot to open a ticket with the operations team. You can also email us at [socialbenefithelp@imaginela.org](mailto:socialbenefithelp@imaginela.org)"
 
-If the user's question is about a referral link below, set canned_response to: \
+# Precedence of canned_response:
+# 1. Coverage list of supported benefit programs
+# 2. Password reset instructions for the Benefit Navigator
+# 3. Benefit Navigator support issue instructions
+# Only if none of the above apply should you consider referral links
```

Comment:
Can this be simplified to:
```suggestion
If canned_response is not set by the above instructions, then consider referral links
```